static/js/composebox_typeahead.js

```diff
@@ -213,6 +213,11 @@ exports.tokenize_compose_str = function (s) {
         i -= 1;
         switch (s[i]) {
             case '#':
+                // If a '#' is immediately followed by a space, cancel the typeahead
+                if (s.charAt(i+1) === " ") {
+                    return '';
+                }
+                // falls through
```

Comment:
I'm not sure this is the best place to do this check; did you consider putting the check in `autocomplete_checks` instead?